tgt/heightmapGenerator.py

```diff
@@ -0,0 +1,25 @@
+'''
+The parent structure that generates heightmaps based
+on user preferences.
+'''
+
+import random
+import noise
+import numpy as np
+
+'''
+This will call the currently best heightmap generator algorithm.
```

Comment:
If we want to follow the python convention, docstrings go inside the function definition as is done [here](https://www.python.org/dev/peps/pep-0257/).